kornia/color/rgb.py

```diff
@@ -141,14 +142,15 @@ def rgba_to_rgb(image: torch.Tensor) -> torch.Tensor:
 
```

Comment:
```suggestion
    References:
        [1] https://en.wikipedia.org/wiki/Alpha_compositing#Alpha_blending
```
I'd add the reference